themes/geekboot/layouts/partials/scripts.html

```diff
@@ -3,9 +3,9 @@
 {{- $jsmap := resources.Get (printf "%s.map" (index (index .Site.Data.assets "main.js") "src")) -}}
 {{- $jsmap.Publish -}}
 
-{{ if eq hugo.Environment "production" -}}
+{{/*  {{ if eq hugo.Environment "production" -}}
```

Comment:
Should we be re-enabling analytics here before merging?

Reply:
Rebased and dropped this commit. thanks!